src/js/components/SelectMultiple/stories/Variant.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+
+import { Box, SelectMultiple, Text } from 'grommet';
+
+const dummyOptions = [
+  'Apple',
+  'Orange',
+  'Banana',
+  'Grape',
+  'Melon',
+  'Strawberry',
+  'Kiwi',
+  'Mango',
+  'Raspberry',
+  'Rhubarb',
+];
+
+export const Variant = () => {
+  const [options, setOptions] = useState(dummyOptions.sort());
```

Comment:
Why do we need to sort these here? Can `defaultOptions` be sorted correctly to begin with?

Reply:
No longer using this code since sort prop was added in latest commit